core-developers/committing.rst

```diff
@@ -80,21 +80,14 @@ to enter the public source tree. Ask yourself the following questions:
 Updating NEWS and What's New in Python
 --------------------------------------
 
-Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
-If the change is particularly interesting for end users (e.g. new features,
-significant improvements, or backwards-incompatible changes), then an entry in
-the ``What's New in Python`` document (in ``Doc/whatsnew/``) should be added
-as well. Changes that affect documentation only generally do not require
-a ``NEWS`` entry.
+Changes that entail NEWS entries
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+Almost all changes made to the code base deserve an entry in ``Misc/NEWS.d``.
```

Comment:
FWIW this is roughly the same wording used by `bedevere`, and during the last sprint we had a discussion about it and ended up tweaking it:
* https://github.com/python/bedevere/issues/439
* https://github.com/python/bedevere/pull/447
* https://github.com/python/bedevere/pull/593

Reply:
Thanks for those links, Ezio. We could align it with the wording used by Bedevere, but I think the current wording is fine.